src/components/screens/dashboard/recentTransactions/recentTransactions.js

```diff
@@ -3,7 +3,7 @@ import { withTranslation } from 'react-i18next';
 import { useSelector } from 'react-redux';
 import { Link } from 'react-router-dom';
 
-import { selectAccount, selectCurrentBlockHeight } from '@store';
+import { selectAccount, selectCurrentBlockHeight } from '@store/selectors';
```

Comment:
Helps import only the selectors and not the other files from `@store` like middlewares and reducers.